tests/integrations/huggingface/cassettes/test_huggingface_inference_client/test_huggingface_document_question_answering_async.yaml

```
@@ -1,4 +1,357 @@
+        5s/G7/6tgr7fGQf6FX5ouF56cAPDcwPvEGTzm3z6VxEs0YZvIqJfboukmuRb1UaGIf5l6eQVln2i
+        f6rEv/YbyBfMub3WtfwrJP9FoZ84Wui10D+A0BeM/mYdf/RegwVQCXz29HRbyoEl7tTJAYi7PXuq
+        CiMxvsEhXg63UaYWdzXF/crH0cZeG/uHkv6qvg81Qj9EHwO637Qg4k5tPaDBsVURy2E83S7tlrWo
+        j0jUh2tEoY68m8S4V8dvhBcQLfNjkvl9ihwxn0Ze0qy4uJelUbY3o8X98cRdzLgPIe1dGndxGdey
+        PU7ZXp8zCTMmJdUOOgrLO/bSKH5O/F/pG1mmPQPINd2UOiC5Fj906kmuBasUaeG+Fm4BsT1XD2OL
+        LdgeSxFc+zrWISC416EOLbj3ILiJu1u7YfR+fdlsYgneGJSa1AIQXoAbzCtkdUagYfUNbgEPzopX
+        y4Dghb/B4rGgznSjZWFmzhXWBPHicxbP1irKgI0shfE3AYdrJq87o7IMOEIMgNPZq2OGIPgB6a8W
+        L7KhDvz/h/aFmMyzCgC0L4TYxW8u/KJpj+IncoDal7wzYvXQXyqLOyDheaY47G6SEDc++EYrg5P3
+        KOF8fLh72PjRcxq7YeLF/p6+UfG7SmAlmDjQmxMEZFu0ZFmJq9HcygTnt/1UN4t13kFW8ixMh/zp
+        bBa/+ddv/j/D9V5OiR8FAA==
+    headers:
+      Accept-Ranges:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)